packages/custom-functions-metadata/package.json

```diff
@@ -32,7 +32,6 @@
   },
   "devDependencies": {
     "@types/assert": "^1.4.2",
-    "@types/commander": "^2.12.2",
```

Comment:
should types/commander in fact be removed here?

Reply:
Yes, yarn gives a warning that "commander" includes types, so "@types/commander" should not be used.